test-projects/pizza-ordering/cli/order/dynamic-async-accepts/dynamic-async-accepts.spec.js

```diff
@@ -0,0 +1,22 @@
+async function dataAllows() {
+	return ['a', 'b', 'c', 'd'];
+}
+
+async function testAllows() {
+	return ['a1', 'b2', 'c3', 'd4'];
```

Comment:
Can we make these more consistent with one another, like:

```suggestion
	return ['a1', 'b1', 'c1', 'd1'];
}

async function testAllows() {
	return ['a2', 'b2', 'c2', 'd2'];
```